packages/x-tree-view/src/internals/models/plugin.ts

```diff
@@ -9,14 +9,33 @@ import { TreeViewItemId } from '../../models';
 import { TreeViewStore } from '../utils/TreeViewStore';
 
 export interface TreeViewPluginOptions<TSignature extends TreeViewAnyPluginSignature> {
+  /**
+   * An imperative api available for internal use.
+   */
   instance: TreeViewUsedInstance<TSignature>;
+  /**
+   * The parameters after being processed with the default values.
```

Comment:
Maybe we can make it more specific that parameters in this context = parameters of the tree view?